infra/hoogle_server.tf

```diff
@@ -115,45 +115,40 @@ NIX_PATH=nixpkgs=https://github.com/NixOS/nixpkgs/archive/c50e680b03adecae01fdd1
 HOOGLE_PATH=$(nix-build --no-out-link -E '(import <nixpkgs> {}).haskellPackages.hoogle')
 mkdir -p /home/hoogle/.local/bin
 ln -s $HOOGLE_PATH/bin/hoogle /home/hoogle/.local/bin/hoogle
-export PATH=/home/hoogle/.local/bin:$PATH
```

Comment:
So the idea is that we just give up on initialization and only do the diff thing? Seems sensible

Reply:
I see it more as using the same script for init and cron, that way they don't get out of sync as they did in #7370. It does rely on the fact that `diff a b` is just as non-zero when `a` and `b` differ as when one doesn't exist.